lib/gui/app/components/svg-icon/svg-icon.tsx

```diff
@@ -95,31 +95,36 @@ export class SVGIcon extends React.Component<SVGIconProps> {
 
 		if (!svgData) {
 			_.find(this.props.paths, relativePath => {
-				// This means the path to the icon should be
-				// relative to *this directory*.
-				// TODO: There might be a way to compute the path
-				// relatively to the `index.html`.
-				const imagePath = path.join(baseDirectory, 'assets', relativePath);
-
-				const contents = _.attempt(() => {
-					return fs.readFileSync(imagePath, {
-						encoding: 'utf8',
+				if (_.startsWith(relativePath, 'http')) {
```

Comment:
Why this changes? SVGIcon is not used in this pull request.

Reply:
Leftover, will remove it